appeals/web/src/server/appeals/appeal-details/site-visit/__tests__/site-visit.test.js

```diff
@@ -18,290 +17,314 @@ describe('site-visit', () => {
 	});
 	afterEach(teardown);
 
-	describe('GET /site-visit/schedule-visit', () => {
-		it('should render the schedule visit page', async () => {
-			const response = await request.get(`${baseUrl}/1${siteVisitPath}${scheduleVisitPath}`);
-			const element = parseHtml(response.text);
+	for (const testType of ['schedule', 'manage']) {
```

Comment:
I can see why you've taken this approach, but in principle we should avoid having logic like this in unit tests. This type of logic increases the risk of bugs in the test code itself, which may either cause false positives, hide issues, etc. If this were application code I would agree that avoiding the duplication is a good idea, but the same doesn't apply to unit tests (it's more important for them to be bulletproof than concise). I think you should remove this and simply duplicate the tests as required. There's some more info here: https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#avoid-logic-in-tests